packages/contracts/hardhat.config.ts

```diff
@@ -82,7 +82,6 @@ if (
   process.env.CONTRACTS_RPC_URL
 ) {
   config.networks[process.env.CONTRACTS_TARGET_NETWORK] = {
-    accounts: [process.env.CONTRACTS_DEPLOYER_KEY],
```

Comment:
This should NOT be deleted. It was originally deleted so that the account funding thing from above would work. But that thing should not exist and it should live inside `hardhat-ovm`. Deleting this line breaks stuff.